### ISSUE TITLE
Winget - suppress autolaunch of vscode after installation completes

### ISSUE THREAD

**Issue by jantari** [microsoft/vscode#125447]
Hi, I'm tagging @lszomoru because I can see you've previously worked on the winget publishing pipeline. Thanks for that!

TLDR: Please change the winget manifests so that VSCode does not autolaunch after being installed

Currently, all of the winget manifests for vscode rely on wingets "generic" or "default" handling of inno installers for installation, and the default arguments passed to the installer appear to come from here: https://github.com/microsoft/winget-cli/blob/cf6f8e58195bf033a1a5872f7b800866cd27a586/src/AppInstallerCommonCore/Manifest/ManifestCommon.cpp#L348

However for VSCode specifically, it is preferable for the UX when the program doesn't seemingly randomly auto-launch after installation, especially since `winget` has/will soon have support for batch-installations of many packages via an `import` feature.

I have already prepared and tested all the changes neccessary to the manifests to suppress the auto-run here in this commit:

https://github.com/jantari/winget-pkgs/commit/3a603f20f32dffa80ff0c28d4ec3eb35652c4de8

however I got the feeling that the release process to the winget-pkgs repo is probably automated so I didn't see a point in going through with a direct PR to winget-pkgs. I could not find any winget publishing CI in this repository directly so I could not make a PR here, instead I'm asking for this change via an issue - hope that's ok.

Basically, all that needs to be done is override the default inno arguments that winget currently uses (which are just `/VERYSILENT` or `/SILENT`) with our own ones. In the manifest that means adding the following block at the root level of the YAML:

    InstallerSwitches:
      Silent: /VERYSILENT /mergetasks=!runcode
      SilentWithProgress: /SILENT /mergetasks=!runcode

See the schema definition here: https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.0.0/manifest.installer.1.0.0.json

PS: I would also appreciate a YAML comment at the top of all winget manifests that states that these manifests are published and maintained via CI and that direct community PRs are "pointless" - this is a popular program so, e.g. if someone feels like they want to add a tag in the future, they might run into the same situation again of trying to PR against what's in the winget-pkgs repo directly

**Comment by lszomoru** [comment]
@jantari, thanks for the great research and write up. Rather than overriding the built-in installer switches I have opted for specifying a `Custom` installer switch which is always passed to the installer by winget. I have updated our publishing pipeline and both Stable and Insiders manifests have been updated.
